upgrade_tests/upgrade_manifest.py

```diff
@@ -30,6 +30,13 @@
 CASSANDRA_5_1 = '5.1'
 TRUNK = CASSANDRA_5_1
 
+CC4 = '4.0.11.'
+CC5 = '5.0.2.'
```

Comment:
every time CC gets rebased, we need to remember to bump this.

maybe there's a better way to identify CC from the version number…